Python/Sandhya/task2/imdb_scraper.py

```diff
@@ -0,0 +1,74 @@
+# Scraping popular, top picks and fan favourite movies/series from imdb
+import requests
+from bs4 import BeautifulSoup
+from pandas import DataFrame
+import yaml
+import os
+
+URL = 'https://www.imdb.com/what-to-watch/'
+categories = ['popular','top-picks','fan-favorites']
+
+class StoreResult:
+    
+    def csvFile(category, all_title_list, all_rating_list):
```

Comment:
```suggestion
    def csv_file(category, all_title_list, all_rating_list):
```
Please use snake case for function names and variable names